### DISCUSSION TITLE
Standard - Legacy mods and items

### DISCUSSION THREAD

**Discussion by annedobalina** [SnosMe/awakened-poe-trade#1310] [Q&A] [answered]
Hi there, just wondering if there was a way to enable legacy items/mods for the players still in Standard?

There are issues searching mods on clusters with Supercharge, Deep Cuts, Wish for Death (but these can be worked around by opening trade and manually entering).

However today's update seems to have completely disabled being able to search for TWWT jewels altogether.

I'm not asking for every single legacy item ever and I know that TWWT is "legacy" but it's barely a week out of Affliction and the Standard market is still flooded with them.

**Comment by SnosMe** [comment, marked as answer]
Will check if I can add it to whitelist.
I've never supported legacy items/mods.

My common response is:
This won't change because I generate data from game files. Data often contains duplicates or WIP/test content that collides with currently actually dropable items/mods. So I filter them by whatever indicates that they're obtainable right now. Without this step I'd need to manually resolve A LOT of conflicts (btw I still do, but only a little), or worst choice is to let random decide which record gets exported.

**Reply by annedobalina** [reply]
You absolute legend, yes please that one item whitelisted would be great for us Standard guys.  

I think I understand the issues around general legacy items and mods (I really appreciate that you whitelisted the legacy life as extra ES craft or explode chests for example).  I don't know if that would mean a bunch of charm/TWWT-specific mods would have issues tho?
